addons/godot-xr-tools/player/player_body.gd

```diff
@@ -171,15 +170,27 @@ func is_class(name : String) -> bool:
 
 # Called when the node enters the scene tree for the first time.
 func _ready():
+	# Set as toplevel means our PlayerBody is positioned in global space.
+	# It is not moved when its parent moves.
+	set_as_toplevel(true)
+
+	# Create our collision shape, sizes will be update later
+	var capsule = CapsuleShape.new()
+	capsule.radius = 1.2
```

Comment:
You may want to just use `player_radius` here' although no-matter what you enter it's going to be overridden by the `_update_player_radius()` call later.

Reply:
I left that out seeing it looked weird that I properly set radius but set height to an arbitrary value while both get updated later on in the setup process to their proper values. Hence the remark `sizes will be update(d) later`, typo in that one

Reply:
Changed it and the comment :)